grammars/hocon.cson

```diff
@@ -120,7 +121,7 @@ repository:
     ]
   ustring:
     comment: "unquoted strings"
-    match: "([^=\\{\\}\\[\\]\\s][^0-9=\\{\\}\\[\\]][^=\\{\\}\\[\\]\\s]*)$"
+    match: "([^:=\\{\\}\\[\\]\\s,][^0-9:=\\{\\}\\[\\],][^=:\\{\\}\\[\\]\\s,]*)"
```

Comment:
This change makes it so that, `:` are treated the same as `=`, commas are not included in unquoted strings, and I've removed the `$` so there can be multiple occurrences on the same line.